core/invoker/src/main/scala/whisk/core/container/ContainerPool.scala

```diff
@@ -512,9 +526,9 @@ class ContainerPool(
             introduceContainer(warmNodejsKey, con)
         }
         info(this, s"Started warm nodejs container $con.id: $con.containerId")
-    } catch {
-        case t: Throwable => warn(this, s"addWarmNodejsContainer encountered an exception: ${t.getMessage}")
-    }
+    }.recover {
+        case t => warn(this, s"addWarmNodejsContainer encountered an exception: ${t.getMessage}")
```

Comment:
Note that this will recover the `Failure` to a `Success` which might (will) be unexpected?

`andThen` would be a better solution to add a log here.